File: Techschool/Techschool.BLL/Models/Vacations/WithoutPayrollVacationFormModel.cs
using System.Collections.Generic;

namespace Techschool.BLL.Models.Vacations
{
    public class WithoutPayrollVacationFormModel : VacationFormModel
    {
        public IEnumerable<WithoutPayrollVacationModel> WithoutPayrollVacations { get; set; }
    }
}

File: Techschool/Techschool/ClientApp/src/app/models/vacations/without-payroll-vacation-form.model.ts
import { WithoutPayrollVacationModel } from './without-payroll-vacation.model';
import { VacationFormModel } from './vacation-form.model';

export class WithoutPayrollVacationFormModel extends VacationFormModel {
  withoutPayrollVacations: WithoutPayrollVacationModel[];
}
